zipline/assets/_assets.pyx

```diff
@@ -42,6 +44,16 @@ from pandas.tslib import normalize_date
 
 CACHE_FILE_TEMPLATE = '/tmp/.%s-%s.v6.cache'
 
+FUTURE_EXCHANGE_MAPPING = {
+    "CFE": "CFE",
+    "CBOT": "CME",
+    "CME": "CME",
+    "COMEX": "CME",
+    "NYMEX": "CME",
+    "ICEUS": "ICE",
+    "NYFE": "ICE"
```

Comment:
Do you think this mapping belongs in zipline, or should the creator of the data have mapped it already?